color_balance/colorimage.py

```diff
@@ -190,26 +144,33 @@ def scale_offset_lut(in_lut, scale=1.0, offset=0):
     min_val = 0
     max_val = len(out_lut) - 1
     logging.info("clipping lut to [{},{}]".format(min_val, max_val))
-    numpy.clip(out_lut, min_val, max_val, out_lut)
-    return out_lut.astype(numpy.uint8)
+    np.clip(out_lut, min_val, max_val, out_lut)
+    return out_lut.astype(np.uint8)
 
 
 def apply_lut(band, lut):
-    '''Changes band intensity values based on intensity look up table (lut)'''
-    if lut.dtype != band.dtype:
-        raise LUTException("Band ({}) and lut ({}) must be the same data " +
-            "type.").format(band.dtype, lut.dtype)
-    return numpy.take(lut, band, mode='clip')
+    """
+    Changes band intensity values based on intensity look up table (lut)
+    """
+
+    # if lut.dtype != band.dtype:
+    #     msg = "Band ({}) and lut ({}) must be the same data type.".format(band.dtype, lut.dtype)
+    #     raise LUTException(msg)
```

Comment:
It would be nice to establish that the check can not be meaningfully applied here and remove these or put in an appropriate check.